docs/assets/css/secondary-nav.less

```diff
@@ -1,85 +1,178 @@
 /* ==========================================================================
    Secondary Nav
+   Creates an expandable show/hide toggle nav on small screens, and a regular left sidebar on large screens
+   Use the following classes on the sidebar container element to add this styling:
+        .content_sidebar
+        .o-expandable
+        .o-expandable__padded
+        .o-expandable__background
+    And use these for child div where the navigation links are:
+        .o-expandable_content 
+        .nav-secondary
    ========================================================================== */
 
-.nav-secondary {
-    margin-bottom: unit(30px / @base-font-size-px, em);
+.content_sidebar {
+    &.o-expandable {
 
-    &_list {
-        margin: 0;
-        padding: 0;
-    }
+        padding: unit( 7.5px / @base-font-size-px, em ) 0px 0px;
```

Comment:
Maybe add a code comment above this to say how the magic number 7.5px is calculated.